ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -445,7 +442,7 @@ Rectangle {
             }
         }
 
-        if (event.matches(StandardKey.Paste)) {
+        if (event.matches(StandardKey.Paste) && QClipboardProxy.hasText) {
```

Comment:
The condition `QClipboardProxy.hasText` doesn't belong here; right below we paste an image

Reply:
done